src/index.js

```diff
@@ -1 +1,19 @@
+import express from 'express'
+import logger from 'morgan'
 
+const app = express()
+const PORT = process.env.PORT || 3000
+
+app.use(logger('dev'))
```

Comment:
Can we put this and the parser stuff in a `middleware/global` folder?  So it does not clutter out `index.js`